x/marketmap/types/codec.go

```diff
@@ -2,14 +2,27 @@ package types
 
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/codec/legacy"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
 // RegisterLegacyAminoCodec registers the necessary x/marketmap interfaces (messages) on the
 // cdc. These types are used for amino serialization.
-func RegisterLegacyAminoCodec(_ *codec.LegacyAmino) {
+func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+	// register the msg-types
+	legacy.RegisterAminoMsg(cdc, &MsgUpdateMarketMap{}, "slinky/x/marketmap/MsgUpdateMarketMap")
+	// legacy.RegisterAminoMsg(cdc, &MsgParams{}, "slinky/x/marketmap/MsgParams")
```

Comment:
Is there a reason this is commented out? im assuming the params handler isnt implemented here?

Reply:
ah no this was waiting for the `Params` PR to be merged.  fixing